app/views/comments/_comment.html.erb

```diff
@@ -10,12 +10,14 @@
       <p><%= comment.body %></p>
     </div>
     <div>
-      <%= link_to edit_comment_path(comment), class: "btn btn-link btn-sm text-muted" do %>
-        <i class="fas fa-edit fa-fw"></i>
-      <% end %>
+      <%if current_user == comment.author %>
```

Comment:
use policies here, instead of having the logic here which would be harder to change later, we can call `policy(comment).edit?`. This will refer to the comment policy if the user is allowed to edit the comment

Reply:
That is very interesting! I didn't think of doing that but that makes sense and shortens up the code!